src/cocotb/types/logic_array.py

```diff
@@ -21,6 +24,22 @@
 if TYPE_CHECKING:  # pragma: no cover
     from typing import Literal
 
+COCOTB_RESOLVE_X: "Literal['error'] | Literal['low'] | Literal['high'] | Literal['random']" = {
+    "VALUE_ERROR": "error",
+    "ZEROS": "low",
+    "ONES": "high",
+    "RANDOM": "random",
```

Comment:
That's the motivation to change the naming here? On a green field, I'd also prefer lower case and `error` over `VALUE_ERROR`, but given where we are today: is the gain big enough to justify the inconsistency between the environment variable and the function argument? I don't think so.

Reply:
I agree- they should share the same set of values. Since we removed and then added it, and it's used a bit differently, I'm on board with renaming the environment variable (COCOTB_LOGIC_RESOLVE ?) and also using the new values. Another reason for renaming is that it's not just X that it resolves. I made them shorter and lower case since they can now be passed in as an arg, not just an environment variable. But if the maintainers want to keep the old names/values, I can switch these back.

Reply:
Maybe a StrEnum?